warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -546,10 +484,60 @@ public void map(K1 key, V1 value, Context context) throws IOException, Interrupt
             context.progress();
         }
         
+        incrementEventCount(value, context);
```

Comment:
I dig the method breakout. This function used to be huge.

Reply:
Thanks!